tsdb/db_test.go

```diff
@@ -2692,3 +2699,38 @@ func TestChunkReader_ConcurrentReads(t *testing.T) {
 	}
 	testutil.Ok(t, r.Close())
 }
+
+func TestTimeMetrics(t *testing.T) {
+	db, closeFn := openTestDB(t, nil, nil)
+	defer closeFn()
+
+	metrics := &dto.Metric{}
+
+	// Check initial values.
+	testutil.Ok(t, db.metrics.minTime.Write(metrics))
```

Comment:
just a reminder that can also use testutil.ToFloat64
"github.com/prometheus/client_golang/prometheus/testutil"

Reply:
Yes, I just copied what was there before.. should I change?

Reply:
doesn't matter, not a big difference.